scripts/server.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -e
+
+ARGS=$*
+
+echo "Run the RWD Docker container."
+
+if [ -z "$RWD_DATA" ]; then
+    echo "Environment variable RWD_DATA not defined."
+    exit 1
+fi
+
+set -x
+
+docker-compose run \
+--rm \
+-e RWD_DATA=$RWD_DATA \
+app
+$ARGS
```

Comment:
Comments from console.sh apply here too. Not sure if $ARGS is relevant and the indentation is off.